src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -33,19 +37,50 @@ object CallActor {
   case object CallAborting extends CallActorState
   case object CallDone extends CallActorState
 
+  /** The `WorkflowActor` will drop `TerminalCallMessage`s that it is unable to immediately process.  This message
+    * represents the acknowledgment from the `WorkflowActor` that the `callMessage` was processed and the status
+    * change has been successfully committed to the database. */
+  final case class Ack(callMessage: WorkflowActor.TerminalCallMessage) extends CallActorMessage
+
+  /** Message to self to retry sending the `callMessage` to the `WorkflowActor` in the absence of an `Ack`. */
+  final case class Retry(callMessage: WorkflowActor.CallMessage) extends CallActorMessage
+
+  /** FSM data class with an optional abort function and exponential backoff. */
+  case class CallActorData(abortFunction: Option[AbortFunction] = None,
+                           backoff: Option[ExponentialBackOff] = None,
+                           timer: Option[Cancellable] = None) {
+
+    /**
+     * Schedule a single retry of sending the specified message to the specified actor.  Return a copy of this
+     * `CallActorData` holding a reference to the timer to allow for its cancellation in the event an `Ack` is
+     * received prior to its expiration.
+     */
+    def copyWithRetry(actor: Actor, callMessage: WorkflowActor.CallMessage)(implicit ec: ExecutionContext): CallActorData = {
+      val timer = actor.context.system.scheduler.scheduleOnce(backoff.get.nextBackOffMillis().millis) {
```

Comment:
Yes.  The only two places this is called are when the retry is being set up and the backoff was just created, and from the retry handler which copies state data that is know to contain a backoff.